### PR TITLE
Add new externals, misc

### LINKED ISSUE #62: Enhancing the ESH copying process
Continuing https://github.com/openhab/openhab-docs/pull/56

> Would it be better to have the whole eclise smarthome docs as a submodule (to make realtions clear to everyone) and to work with symlinks (if jekyll and friends like that)? Without symlink an idea would be to have an sh script (like you have now) copying files with an additional step, marking them as copied. An added YAML front matter "page.source" in the head of the file would be a nice approach I guess.
> 
> And to answer your question, yes I think so, we should copy as much as possible but add openhab specific things on top. A styles2.css loaded after style.css could be a place for everything orange and openhab'y.

---

> to work with symlinks (if jekyll and friends like that)?

I think this won't work, at least I wasn't very successful when I experimented with such a solution.
What we will indeed need sooner or later is a better manageable copy mechanism, doing it in the pom.xml as it is now will definitely become too complex at some point in time.

> I think we should copy as much as possible

Feel free to adapt the copy mechanism accordingly.

---

Will have to find the time for that.
##

### LINKED ISSUE #481: Add Alexa to the docs
Alexa has a "skill" for openHAB (jeah).
https://community.openhab.org/t/official-alexa-smart-home-skill-for-openhab-2/23533
Should we take that to the docs too?

### LINKED ISSUE #495: Mycroft skill
An openHAB [Mycroft](https://mycroft.ai/) skill is available:
https://community.openhab.org/t/mycroft-openhab-skill/33475
Should be integrated in the [System Integration](http://docs.openhab.org/addons/io.html) page?
Thank you.

### PR DESCRIPTION
**Pull request by ThomDietrich**
@kaikreuzer @mortommy @mboremski @ghys

Closes #62 
Resolves #372
Resolves #481
Resolves #495  
Resolves #505 
Depends https://github.com/openhab/org.openhab.ui.habpanel/issues/222

I missed this one: https://github.com/openhab/openhab-docs/issues/480

This PR adds:

* openHABian USAGE
* alexa-skill USAGE
* mycroft-skill USAGE
* HABPanel config article
* tidy-maven-plugin

Plus many small improvements

### CODE REVIEW

File: addons/uis.md [hunk 1]
```diff
@@ -21,6 +23,13 @@ title: User Interfaces
 | [iOS App]({{docu}}/addons/uis/apps/ios.html) | The native iOS app to access openHAB on the go. |
 | [Windows 10 App]({{docu}}/addons/uis/apps/windows.html) | The native Windows 10 app to access openHAB on the go. |
 
+## Iconsets
+
+Basic UI, Classic UI and all apps present a graphical user interface as defined in [Sitemaps]({{base}}/configuration/sitemaps.html) by the help of [Items]({{base}}/configuration/items.html).
```
Comment (bgilmer77) [inline addons/uis.md]: Consider, "Basic UI, Classic UI and all **openHAB** apps present a graphical user interface as defined in Sitemaps**, with ** the help of Items.

File: addons/uis.md [hunk 2]
```diff
@@ -21,6 +23,13 @@ title: User Interfaces
...
+Each Item respectively each Sitemap element may be assoziated with an icon visible next to it.
```
Comment (bgilmer77) [inline addons/uis.md]: I do not understand what you mean by this sentence - consider rewording it.
Consider changing "assoziated" with "associated".

File: addons/uis.md [hunk 3]
```diff
@@ -21,6 +23,13 @@ title: User Interfaces
...
+The Classic Icons Set is currently the only set of icons provided.
```
Comment (bgilmer77) [inline addons/uis.md]: Consider changing to, "Currently, only the Classic Icon Set is provided."

**Comment by ThomDietrich** [comment]
Thanks @bgilmer77!

@kaikreuzer could you please have a final look at the changes to `pom.xml` introduced through the tidy plugin (Motivation: the file had a few formatting errors when I first touched it) If you dislike the addition, we can remove it.
Afterwards please feel free to merge and issue a new content update. I'm on standby if anything comes up. Thanks!

**Comment by ThomDietrich** [comment]
@kaikreuzer just in case you missed the mentioning. If you don't have the time to do this prior to the SHD, the PR can just as well wait till next week.

**Comment by kaikreuzer** [comment]
Hey @ThomDietrich, I am afraid you are right, I completely missed the mention here, sorry :-(

Just tested it and I see the meta-data on the headers being rendered on all add-ons pages:

screen shot 2017-10-18 at 16 01 40

Am I doing anything wrong?

**Comment by ThomDietrich** [comment]
😲 weird.

Seems like the front matter part is rendered which I would guess happens because jekyll didn't identify it as that. Any obvious errors in the md file? I'll fire up my Vagrant machine...

**Comment by ThomDietrich** [comment]
Just quickly went through the PR changes and don't think that any of them would have that effect.
Did you test this in your normal environment? I wonder if the issue is related to some weird encoding/lineendings quirk

**Comment by ThomDietrich** [comment]
Tested, not seeing the issue.

1. Checkout the PR
2. Execute update-external-resources.sh
3. Execute jekyll serve
4. Success.

The only problem I encountered was that the iconset once again failed to build due to the missing data folder (commit not included in PR, no issue)

**Comment by kaikreuzer** [comment]
Maybe an issue with my Jekyll version... Anyhow, the rest looks good, so let's assume that Github does it correctly.

**Comment by kaikreuzer** [comment]
Looks all good! 👍

**Comment by ThomDietrich** [comment]
Yep, looks good. Thanks!

**Pull request merged**